desktop/libs/aws/src/aws/s3/__init__.py

```diff
@@ -132,6 +132,8 @@ def s3datetime_to_timestamp(datetime):
   # There is chance (depends on platform) to get
   # `'z' is a bad directive in format ...` error (see https://bugs.python.org/issue6641),
   # but S3 always returns time in GMT, so `GMT` and `.000Z` can be pruned.
+  if datetime.count('GMT') == 2:                 # workaround for ozone s3 connector
```

Comment:
Suggestion: Why is this workaround required? Let's add a more verbose comment explaining the reasoning?

Reply:
Yes, I will add more details, Working on E2E testing

Reply:
Is there a way to distinguish regular S3 or S3-Ozone?

This should be a test case for regular S3 and S3-Ozone for regression testing.